packages/suite/src/views/wallet/send/index.tsx

```diff
@@ -59,15 +60,21 @@ const Send: React.FC = ({ children }) => {
         metadataEnabled: state.metadata.enabled && !!state.metadata.provider,
     }));
 
-    const { selectedAccount } = props;
+    const { selectedAccount, coinjoinAccounts } = props;
 
     if (selectedAccount.status !== 'loaded') {
         return <WalletLayout title="TR_NAV_SEND" account={selectedAccount} />;
     }
 
+    // find coinjoin account related to current account
+    const coinjoinAccount =
```

Comment:
I've created [a selector for that](https://github.com/trezor/trezor-suite/blob/302d392e8e6463e34aa4290954f663322b4a4371/packages/suite/src/reducers/wallet/coinjoinReducer.ts#L141), would make sense to use it here

Reply:
yeah, why not, but who will be first? chicken or egg? :)
im suggesting to add this selector in your PR